client/src/components/PersonCard/personcard.css

```diff
@@ -0,0 +1,64 @@
+.PersonCard {
+width: 229px;
+height: 289px;
+border: 1px solid var(--yellow);
+background: var(--white);
+position: absolute;
+left: 0%;
+right: 0%;
+top: 0%;
+bottom: 0%;
+box-sizing: border-box;
+box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.PersonCard__Img {
+position: absolute;
+width: 150px;
+height: 150px;
+left: 35px;
+top: 12px;
+}
```

Comment:
Could you use flexbox to layout the items inside the `PersonCard`? Also, the modifier or element should be lower case (i.e. `.PersonCard__image`)